code/_onclick/hud/screen_objects.dm

```diff
@@ -135,6 +144,43 @@
 		else
 			icon_state = icon_empty
 
+/obj/screen/inventory/proc/update_overlays()

```

Comment:
This proc name doesn't work well if this proc always adds the overlay. The behavior should probably be changed, or the name should be changed.